website/static/js/nodeControl.js

```diff
@@ -174,7 +174,11 @@ var ProjectViewModel = function(data, options) {
     };
 
     self.hasIdentifiers = ko.pureComputed(function() {
```

Comment:
If this function will only be checking for the existence of a DOI, let's go ahead and rename it to `hasDOI` for clarity.